### ISSUE TITLE
Prevent death make unconscious units nearly impossible to kill

### ISSUE THREAD

**Issue by alganthe** [acemod/ACE3#839]
ACE3 Version: 2015-04-24: build around 14h UTC+1

Mods:

* @cba_a3
* @ace3

Placed ACE3 Modules:
Medical settings:
```
Medical level: Basic
Enabled for: Players only
Enable advanced wounds: No
Vehicle crashes: Yes
Player damage: 1
AI damage: 1
AI unconsciousness DISABLED
Prevent instant death YES
Bleeding coef: 1
Sync status: YES
```

**Description:**
People or AI affected by the Prevent death effect just flop on the ground and are practically impossible to kill when you shoot them, they will eventually bleed out but this is not normal behavior.

**Steps to reproduce:**
Place the module with the same options
place a player NATO rifleman with "this setCaptive true;" to avoid getting shot
place a baddie
shoot the baddie
enjoy seeing the baddie flop on the ground as you unload mag after mag into him

**Where did the issue occur?**
SP and editor didn't tested in MP

**RPT log file:**
Nothin' poppin'

**Comment by gienkov** [comment]
I think it should only put them once to unconcious if damage dealt was enough to one shot/explosion kill and further damage should deal damage normally and kill.
I do not like this option much tho, you can get hit by GBU-12 and still be able to be healed by medic.

**Comment by alganthe** [comment]
problem being they just flop on the ground after taking gbu's after gbu's.

**Comment by Glowbal** [comment]
From reading this, prevent death does exactly what it name implies; it prevents the death of a unit. And when making use of the timer, it does so until that timer runs out. This is also how it was designed; it's a basic revive system.

If I understand you correctly, you are asking for a 'limit death' setting; where a unit won't be killed right away but it is still possible if shot again?

**Comment by gienkov** [comment]
When I first saw that option in AGM I had above in mind. I tought it will put unit into unconsciousness regardless of damage dealt but after that unit will be easy to kill if hitted again.

**Comment by alganthe** [comment]
The problem being that after a certain threshold of damage the unit go unconscious, after that the unit can take ANY amount of damage and not die
that's the issue

**Comment by Glowbal** [comment]
That is because you are using the setting `prevent death`, which does exactly what it says on the tin.

We might look at adding a `prevent instant death` setting and see if our naming of the settings are descriptive enough. @KoffeinFlummi

**Comment by alganthe** [comment]
The module description definitly say prevent INSTANT death

**Comment by bux578** [comment]
If you haven't defined it anywhere else the setting is called `preventInstaDeath`

https://github.com/acemod/ACE3/blob/master/addons/medical/CfgVehicles.hpp#L120

**Comment by alganthe** [comment]
http://puu.sh/hpaCt/291815f621.png 
screenshot of the module

**Comment by Glowbal** [comment]
Thanks @bux578. I will adjust this accordingly tonight.

**Comment by kripto202** [comment]
I think prevent instant death should be limited too. Anywhere on the body except for the head should prevent instant death. It makes sense since you die when you get shot in the head basically.

**Comment by J-Ziegler** [comment]
I would disagree. Preventing instant death should do exactly that, for those groups who prefer a less punishing system when it suits them. Any more advanced customization should be an alternate configuration setting. The team obviously has the final decision when making design choices, but now that AGM and CSE have been discontinued I would suggest that versatility is important.

**Comment by alganthe** [comment]
I personally think that preventing instant death as it was done in AGM is pretty much perfect.
The units didn't got instagibbed but if a whole group of ai was hammering you it was highly probable that you would've still died because,well a group of ai don't stop to fire directly.

it was pretty much a safe method to avoid 500m headshots from ai standing being shot at.

an other drop down menu below the Prevent instant death one would be better like
```
Prevent instant death  * yes / no
Unconscious unit invincible * yes / no

**Comment by gienkov** [comment]
:+1:

**Comment by Glowbal** [comment]
#861

**Comment by alganthe** [comment]
now waiting for a merge

**Comment by Glowbal** [comment]
> now waiting for a merge
> tested and definitely fixed in #861

Well it isn't implemented, besides the fact there is a setting available for it..

**Comment by Glowbal** [comment]
It's now implemented.
